cheatsheets/XML_Security_Cheat_Sheet.md

```diff
@@ -687,7 +687,7 @@ When the definition of an element `A` is another element `B`, and that element `
 
 #### Quadratic Blowup
 
-Instead of defining multiple small, deeply nested entities, the attacker in this scenario defines one very large entity and refers to it as many times as possible, resulting in a quadratic expansion `(O(n2))`.
+Instead of defining multiple small, deeply nested entities, the attacker in this scenario defines one very large entity and refers to it as many times as possible, resulting in a quadratic expansion (<i>O(n<sup>2</sup>)</i>).
```

Comment:
Fixed, it looks like this: 
![image](https://user-images.githubusercontent.com/16072586/157517301-d7ae6f43-fa6f-4979-a54a-1153af97f659.png)

Reply:
I can't use inline HTML (linter rule) - I'll go for something else then